### ISSUE TITLE
run searchable.py , show no attribute 'search_all'

### ISSUE THREAD

**Issue by Sue1990** [MISP/PyMISP#736]
If I executed searchable.py, it will show below down messages:

File "searchall.py", line 41, in <module>
    searchall(misp, args.search, args.quiet, misp_url, args.output)
  File "searchall.py", line 16, in searchall
    result = m.search_all(search)
AttributeError: 'PyMISP' object has no attribute 'search_all'

do I miss import ?

**Comment by kartone** [comment]
Same issue.

**Comment by Rafiot** [comment]
Right, I need to update this example, see: https://github.com/MISP/PyMISP/issues/712#issuecomment-790612121
